public/index.js

```diff
@@ -175,7 +179,7 @@ paperPlane.addEventListener("click", () => {
   let emailOBject = {
     [userEmail.name]: userEmail.value,
   };
-  fetch("public/server.js", {
```

Comment:
Well spotted fix for the error message we were getting in the console. @josueJURE